manifest.json

```diff
@@ -1,23 +1,22 @@
 {
     "manifest_version": 2,
-    "content_scripts": [{
-        "js":               ["float.js"],
-        "matches":          ["http://steamcommunity.com/market/listings/730/*",
-                            "https://steamcommunity.com/market/listings/730/*",
-                            "http://*.steamcommunity.com/market/listings/730/*",
-                            "https://*.steamcommunity.com/market/listings/730/*"]
-    }],
-    "permissions": [
-        "http://steamcommunity.com/market/listings/730/*",
-        "https://steamcommunity.com/market/listings/730/*",
-        "http://*.steamcommunity.com/market/listings/730/*",
-        "https://*.steamcommunity.com/market/listings/730/*"
+    "name": "CSGOFloat Market Checker",
+    "short_name": "CSGOFloat",
+    "version": "1.1.0",
+    "description": "Dedicated API for fetching the float value and paint seed of CSGO items on the market",
+    "icons": {
+        "16": "icons/16.png",
+        "48": "icons/48.png",
+        "128": "icons/128.png"
+    },
+    "content_scripts": [
+        {
+            "matches": ["*://*.steamcommunity.com/market/listings/730/*"],
+            "js": ["float.js"]
+        }
     ],
-    "icons": {"16": "icons/16.png",
-              "48": "icons/48.png",
-              "128": "icons/128.png"},
-    "description":  "Dedicated API for fetching the float value and paint seed of CSGO items on the market",
-    "name":         "CSGOFloat Market Checker",
-    "short_name":   "CSGOFloat",
-    "version":      "1.0.2"
-}
\ No newline at end of file
+    "permissions": [
+        "https://api.csgofloat.com:1738/*",
```

Comment:
I think it's necessary for Firefox (WebExtensions).
From [the MDN page](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/permissions):
> Host permissions are specified as match patterns, and each pattern identifies a group of URLs for which the extension is requesting extra privileges.
The extra privileges include:
    * XHR access to those origins